plugins/modules/ec2_eni_info.py

```diff
@@ -17,10 +17,17 @@
 author: "Rob White (@wimnat)"
 requirements: [ boto3 ]
 options:
+  eni_id:
+    description:
+      - The ID of the ENI.
+      - This option is mutually exclusive of I(filters).
+    type: str
+    version_added: 1.2.0
```

Comment:
```suggestion
    version_added: 1.3.0
```
Same for other new options